### PR TITLE
Added new study

### PR DESCRIPTION
**Pull request by amapeters**

### CODE REVIEW

File: synapseAnnotations/data/neuro.json [hunk 1]
```diff
@@ -341,6 +341,10 @@
     "columnType": "STRING", 
     "maximumSize": 250, 
     "enumValues": [
+      {
+        "value": "iPSCAstrocytes",
```
Comment (kdaily) [inline synapseAnnotations/data/neuro.json]: could it be "iPSC Astrocytes"? easier to read?

Comment (kdaily) [reply synapseAnnotations/data/neuro.json]: Also, missing a closing brace.

**Comment by ychae** [comment]
I'm fairly certain it's already been used. Checking now.

**Comment by ychae** [comment]
@kdaily  Can confirm, the term `iPSCAstrocytes` is being used on all the files in this study: https://www.synapse.org/#!Synapse:syn10344967

**Comment by ychae** [comment]
Merged!

**Pull request merged**